example/book_store/lib/main.dart

```diff
@@ -53,14 +53,14 @@ final routeMap = RouteMap(
           validate: (info, context) => booksDatabase.books.any(
             (book) => book.id == info.pathParameters['id'],
           ),
-          child: MaterialPage(
+          pageBuilder: () => MaterialPage(
```

Comment:
Should it just be `builder` since there is only one builder?

Reply:
Good question - my thinking with `pageBuilder` was to make it clear it needs to return a page and not a widget. But I guess it wouldn't compile anyway, and it should be pretty obvious from the error

Reply:
Yes, I was thinking that the code wouldn't run since the developer would get an error.

Reply:
Done